src/mgr/ActivePyModules.cc

```diff
@@ -432,6 +432,28 @@ PyObject *ActivePyModules::get_python(const std::string &what)
       mgr_map.dump(&f);
     });
     return f.get();
+  } else if (what == "active_clean"){
```

Comment:
spacing looks odd on github (not sure if it's right) but the logic looks correct